charts/epinio/100.0.0+up1.2.1/Chart.yaml

```diff
@@ -0,0 +1,44 @@
+annotations:
+  artifacthub.io/license: Apache-2.0
+  catalog.cattle.io/auto-install: epinio-crd=match
+  catalog.cattle.io/certified: rancher
+  catalog.cattle.io/display-name: Epinio
+  catalog.cattle.io/experimental: "true"
+  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.25.0-0'
+  catalog.cattle.io/namespace: cattle-epinio-system
+  catalog.cattle.io/permits-os: linux,windows
+  catalog.cattle.io/rancher-version: '>= 2.6.0-0 < 2.7.0-0'
+  catalog.cattle.io/release-name: epinio
+  catalog.cattle.io/type: cluster-tool
+  catalog.cattle.io/upstream-version: 19.0.3
```

Comment:
is this correct?

Reply:
That would be a question for the Epinio team. Maybe @andreas-kupries or @thehejik?

Reply:
@PennyScissors 

If the 19.0.3 refers to the epinio chart version then this is wrong. Should be `1.2.1`.
See also `100.0.0+up1.2.1` in the file name. 

I see this bad version in the #1814 as well. I do not remember writing it, anymore. I suspect that this is a copy/paste error I did. I.e. that I copied the annotations from some example or existing chart and then missed editing this one to the correct version number.

Apologies for that.

As this (2126) is your PR I suspect that I will not be able to fix that, right ? (I.e. no permissions)

Reply:
@andreas-kupries , I thought we were all good to release? I'm confused between this message and the Slack messages. Can we merge this as is?

Reply:
I thought that we were good to release. I got very much blind-sided by the issue above.

If a bogus `upstream-version` like the above is a blocker (*) then we are not good to release, and should fix that annotation in the modified chart. Should (can?) this be fixed in this PR ? Or should I create a PR fixing it and then @PennyScissors updates this here with it ?

---
(*) I do not know what Rancher (marketplace) considers a blocker vs not.

Reply:
To work ahead a bit, PR #2135 is a dev-v2.6 PR which fixes the annotation in the chart, and only that.
I hope that making it was the right thing to do.

Reply:
I think the call for whether the upstream version should be a blocker for release is up to the Epinio team! 

From the charts side, we have approved the PR and are just waiting for your approval on the PR review @andreas-kupries before merging.